sdk/formrecognizer/Azure.AI.FormRecognizer/src/ContentType1.cs

```diff
@@ -1,22 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-using Azure.Core;
-
-namespace Azure.AI.FormRecognizer
-{
-    [CodeGenModel("ContentType1")]
-    internal enum ContentType1
```

Comment:
Originally we had an internal `ContentType1` and a public `ContentType`. The internal type was used to build the internal models to send to the service, while the public one was part of the API we expose to users. They were the same thing so we just needed to translate from one to another when sending a request. I don't know exactly what made CodeGen create this `ContentType1`, but now it correctly uses the public `ContentType`.

Reply:
`ContentType1` was created because the service added `bmp` type to all but one endpoint and that created 2 different `ContentType` types. 
With the latest update, the service is now supporting `bmp` in all their endpoints so no need for special variables.

For more context, when the service did this, they broke most generators, so codegen team had to make changes for the SDKS to be able to have different `ContentTypes` :)